central/networkgraph/flow/datastore/internal/store/postgres/store.go

```diff
@@ -98,12 +98,12 @@ const (
 	pruneNetworkFlowsSrcStmt = `DELETE FROM %s child WHERE NOT EXISTS
 		(SELECT 1 from deployments parent WHERE child.Props_SrcEntity_Id = parent.id::text AND parent.clusterid = $1)
 		AND Props_SrcEntity_Type = 1
-		AND LastSeenTimestamp < $2`
+		AND updatedat < $2`
```

Comment:
As @mtodor mentioned maybe we should keep old behaviour and combine it with the new one with a higher (e.g. 2 times) window.
+ validation around window size – too small (< 10 min) could cause issues with pruning here.

Reply:
Discussed here: https://github.com/stackrox/stackrox/pull/14483#discussion_r1995744799